src/options/options.h

```diff
@@ -241,7 +241,7 @@ class CVC4_PUBLIC Options {
   unsigned getParseStep() const;
   unsigned getThreadStackSize() const;
   unsigned getThreads() const;
-
+  bool getHoFlattenTotal() const;
```

Comment:
I would prefer if this was not a parsing option. Can you instead implement this as a rewrite internally after parsing?

Reply:
I'll purge it from this PR and add in another one then. This option is actually not really impactful, although sometimes it helps